README.md

```diff
@@ -553,6 +553,9 @@ IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2019.
 - <a name="Stoffregen19iccv"></a>Stoffregen, T., Gallego, G., Drummond, T., Kleeman, L., Scaramuzza, D.,  
 *[Event-Based Motion Segmentation by Motion Compensation](http://rpg.ifi.uzh.ch/docs/ICCV19_Stoffregen.pdf)*,  
 IEEE Int. Conf. Computer Vision (ICCV), 2019. [PDF (animations best viewed with Acrobat Reader)](http://rpg.ifi.uzh.ch/docs/ICCV19_Stoffregen.pdf), [YouTube](https://youtu.be/0q6ap_OSBAk)
+- <a name="almatrafi2019davis"></a>Almatrafi, M. M.,  Hirakawa, K.,  
+*[DAViS Camera Optical Flow](http://doi.org/10.1109/TCI.2019.2948787)*,  
+IEEE Transactions on Computational Imaging (TCI), 2019.
```

Comment:
Thanks.
Journal abbreviation: http://ieeeauthorcenter.ieee.org/wp-content/uploads/Journal-Titles-and-Abbreviations.pdf
So, IEEE Trans. Comput. Imag.
BTW, is there a link to a post-print (open access) version of the paper?